seqr/management/commands/resend_welcome_email_tests.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from django.core.management.base import CommandError
+
+TEST_USER_EMAIL = 'test_user@test.com'
+REFERRER_EMAIL = 'test_user_no_staff@test.com'
+
+
+class ResendWelcomeEmailTest(TestCase):
+    @mock.patch('seqr.utils.communication_utils.send_welcome_email')
+    @mock.patch('django.contrib.auth.models.User')
```

Comment:
Never mock django models. Just make your test work with the fixture data

Reply:
Updated